components/02-molecules/pager/pager.twig

```diff
@@ -32,9 +32,9 @@
 {% set pager__base_class = 'pager' %}
 
 {% if items %}
-  <nav {{ bem(pager__base_class) }} role="navigation" aria-labelledby="pagination-heading">
-    <h4 id="pagination-heading" {{ bem('visually-hidden') }}>{{ 'Pagination'|t }}</h4>
-  <ul {{ bem('items', [], pager__base_class, ['js-pager__items']) }}>
+  <nav {{ bem(pager__base_class) }} role="navigation" aria-labelledby="pagination-heading{{ pager__uid }}">
```

Comment:
maybe add dashes between the text and the variable? `pagination-heading-{{ pager__uid }}`
So it's
`pagination-heading-1`
instead of the current
`pagination-heading1`

Reply:
Didn't make a difference - I'm fine leaving this